hasura/migrations/default/1652889910989_init-omni/up.sql

```diff
@@ -0,0 +1,329 @@
+CREATE SCHEMA "omni";
+
+CREATE TYPE "omni"."products_stage" AS ENUM (
```

Comment:
Instead of using postgres native enums, we should use enum tables. Hasura docs has an explanation why: https://hasura.io/docs/latest/graphql/core/databases/postgres/schema/enums/